introduction.md

```diff
@@ -0,0 +1 @@
+Hello my name is... Whhaaa my name is...... checka checka slim shady
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.